openshift/templates/deploy.dc.yaml

```diff
@@ -72,6 +87,10 @@ objects:
           containers:
             - image: ${DOCKER_REGISTRY}/${PROJECT_TOOLS}/${NAME}:${SUFFIX}
               imagePullPolicy: Always
+              volumeMounts:
+                - name: ${NAME}-${SUFFIX}
+                  mountPath: /app/src/utils/keycloak.json
```

Comment:
Can you move 3 variables stored in the `keycloak.json` into `.env` file and use them just like how we use other env variables. I don't see any reason that they are being treated differently.

Reply:
@Kyubinhan I dont want to introduce more environment variables. Also please go through https://www.keycloak.org/docs/latest/securing_apps/#_javascript_adapter as I am using same concept.

Reply:
I don't see anywhere not to use environment variables in this docs. You can still initiate keycloak instance the way you want, but I am suggesting to reference env variables because I don't want to add a file coming from a configmap in build time later when we go production mode. You can mount it like this `/app/src/utils/keycloak.json` because we are running the app in dev mode

Reply:
@Kyubinhan the config map will be mounted during deployment time rather build time. Also I have already suggested that there may be more params coming in as part of Keycloak config and I dont want whole lot of env variables floating everywhere.

Reply:
Currently yes the config map is mounted during deployment, but your strategy won't work if we build the app into static files during the build time when we go to prod using caddy. Take a good look at Devhub's DC and BC because this is what we are aiming for. https://github.com/bcgov/devhub-app-web/blob/master/openshift/templates/dc.yaml